katas/content/distinguishing_states/media/Task201_Rotation.svg

```diff
@@ -0,0 +1 @@
0,137.115 136.821 C 137.179 136.758,137.176 136.706,137.108 136.706 C 137.041 136.706,136.619 136.553,136.171 136.366 C 135.723 136.179,135.357 136.076,135.357 136.138 M139.404 136.576 C 140.031 136.954,140.523 137.052,140.180 136.731 C 140.062 136.622,139.713 136.466,139.404 136.385 L 138.842 136.238 139.404 136.576 M137.830 137.232 C 138.139 137.406,138.645 137.616,138.954 137.698 L 139.517 137.847 138.954 137.529 C 138.645 137.355,138.139 137.145,137.830 137.063 L 137.268 136.914 137.830 137.232 M140.978 137.232 C 141.287 137.406,141.793 137.616,142.102 137.698 L 142.664 137.847 142.102 137.529 C 141.793 137.355,141.287 137.145,140.978 137.063 L 140.416 136.914 140.978 137.232 M139.629 137.960 C 139.629 138.112,140.797 138.536,140.908 138.425 C 140.959 138.374,140.692 138.224,140.315 138.093 C 139.938 137.961,139.629 137.901,139.629 137.960 M142.901 138.030 C 143.018 138.140,143.367 138.296,143.676 138.376 L 144.238 138.524 143.676 138.185 C 143.050 137.807,142.557 137.709,142.901 138.030 M142.102 139.050 C 142.729 139.427,143.221 139.525,142.878 139.205 C 142.761 139.095,142.411 138.939,142.102 138.858 L 141.540 138.711 142.102 139.050 M145.250 139.050 C 145.877 139.427,146.369 139.525,146.026 139.205 C 145.908 139.095,145.559 138.939,145.250 138.858 L 144.688 138.711 145.250 139.050 M143.676 139.722 C 143.985 139.888,144.491 140.086,144.800 140.163 C 145.110 140.240,144.958 140.108,144.463 139.870 C 143.446 139.381,142.815 139.262,143.676 139.722 M146.824 139.705 C 147.133 139.880,147.639 140.090,147.948 140.171 L 148.510 140.320 147.948 140.003 C 147.639 139.828,147.133 139.618,146.824 139.536 L 146.262 139.388 146.824 139.705 M145.475 140.433 C 145.475 140.585,146.643 141.009,146.754 140.898 C 146.805 140.847,146.538 140.698,146.161 140.566 C 145.784 140.435,145.475 140.375,145.475 140.433 M148.623 140.433 C 148.623 140.585,149.791 141.009,149.902 140.898 C 149.953 140.847,149.686 140.698,149.309 140.566 C 148.932 140.435,148.623 140.375,148.623 140.433 M151.877 140.997 C 151.760 141.366,151.601 141.439,151.131 141.340 C 150.536 141.215,150.535 141.216,151.066 141.480 C 151.514 141.703,151.645 141.679,151.902 141.328 C 152.274 140.819,152.542 141.000,153.832 142.628 C 154.306 143.227,154.694 143.622,154.694 143.507 C 154.694 143.391,154.433 143.002,154.115 142.643 C 153.797 142.284,153.280 141.661,152.965 141.259 C 152.296 140.406,152.081 140.355,151.877 140.997 M147.948 141.504 C 148.257 141.679,148.763 141.888,149.073 141.970 L 149.635 142.119 149.073 141.801 C 148.763 141.627,148.257 141.417,147.948 141.335 L 147.386 141.186 147.948 141.504 M149.747 142.226 C 149.747 142.294,150.045 142.454,150.409 142.581 C 150.985 142.782,151.045 142.881,150.872 143.338 C 150.607 144.034,150.851 144.104,152.782 143.885 L 154.356 143.706 152.714 143.774 L 151.072 143.843 151.140 143.271 C 151.195 142.809,151.069 142.642,150.478 142.400 C 150.076 142.236,149.747 142.157,149.747 142.226 " stroke="none" fill="#ec7c7c" fill-rule="evenodd"></path></g></svg>
```

Comment:
This file is massive for what is not a particularly high fidelity or complexity image. The SVG should be reworked to be minimal.